### PR TITLE
Unique runner ID

### PR DESCRIPTION
**Pull request by lapineige**

### CODE REVIEW

File: doc/POST_INSTALL.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-This is a dummy disclaimer to display after the install
+The app url is `__DOMAIN____PATH__`. You will not need this in the future, the runner will contact Peertube directly.
 
-The app url is `__DOMAIN____PATH__`
+The app install dir is `__INSTALL_DIR__`.
 
-The app install dir is `__INSTALL_DIR__`
+The app id is `__ID__`.
```
Comment (alexAubin) [inline doc/POST_INSTALL.md]: This is confusing and looks like the dummy / example notification.md ... If this ain't useful, then no need to keep it around x_x

File: scripts/install [hunk 2]
```diff
@@ -62,8 +59,15 @@ ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$ap
 #=================================================
 # RUNNER REGISTRATION
 #=================================================
+ynh_script_progression --message="Registering the runner on Peertube server..." --weight=1
+
+runner_id = ynh_string_random --length=5 --filter='0-9'
```
Comment (alexAubin) [inline scripts/install]:
```suggestion
runner_id=$(ynh_string_random --length=5 --filter='0-9')
```

File: scripts/remove [hunk 3]
```diff
@@ -10,10 +10,13 @@ source _common.sh
 source /usr/share/yunohost/helpers
 
 #=================================================
-# RUNNER REGISTRATION
+# RUNNER UNREGISTRATION
 #=================================================
+ynh_script_progression --message="Unregistering the runner on Peertube server..." --weight=1
 
-peertube-runner unregister --url runner_url --registration-token runner_token --runner-name $app$(hostname)
+runner_id = ynh_app_setting_get --app=$app --key=runner_id
```
Comment (alexAubin) [inline scripts/remove]: You don't need to fetch the runner_id setting, this is app v2 format, all settings are automatically available in scripts

Comment (alexAubin) [inline scripts/remove]: You don't need to fetch the runner_id setting, this is app v2 format, all settings are automatically available in scripts

File: scripts/restore [hunk 4]
```diff
@@ -58,6 +58,17 @@ systemctl enable $app.service --quiet
 
 yunohost service add $app --description="Peertube remote runner" --log="/var/log/$app/$app.log"
 
+#=================================================
+# RUNNER REGISTRATION
+#=================================================
+
+ynh_app_setting_get --app=$app --key='runner_id'
```
Comment (alexAubin) [inline scripts/restore]: (same as for remove, no need to load runner_id explicitly)

File: doc/POST_INSTALL.md (hunk 1 above)
Comment (lapineige) [reply doc/POST_INSTALL.md]: As this app is not installed in a classical way, and as you need to register those parameters in Peertube after installing this app, I thought it was best to remind them to the users.

(The ID is here to differentiate between several runners, if applicable)

File: scripts/install (hunk 2 above)
Comment (lapineige) [reply scripts/install]: Thanks a lot !

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
!bookwormtestme

**Comment by lapineige** [comment]
I forgot CI is useless here.

**Pull request merged**